test-apps/display-test-app/src/backend/DtaElectronMain.ts

```diff
@@ -48,7 +48,7 @@ const dtaElectronMain = async () => {
 
   const opts = {
     webResourcesPath: path.join(__dirname, "..", "..", "build"),
-    iconName: "display-test-app.ico",
+    iconName: "../public/display-test-app.ico",
```

Comment:
do we need this to point to public?
should it get auto picked up now?

Reply:
removed the relative pathing in front of the `.ico` file, confirmed works